config/manager/manager.yaml

```diff
@@ -70,10 +70,8 @@ spec:
         - /manager
         args:
         - --leader-elect
-        env:
-          - name: CLUSTER_NAME
-            value: CLUSTER_NAME_VALUE # Replace it with the cluster name where numaplane will get deployed.
         image: numaplane-controller:latest
+        imagePullPolicy: IfNotPresent
```

Comment:
did this get changed by accident after local testing?

Reply:
yes, I definitely need a new file for local testing .